conftest.py

```diff
@@ -75,6 +75,10 @@ def empty_deployment():
         for _ in kcidb.mq.EmailSubscriber(project, topic, subscription). \
                 pull_iter(timeout=30):
             pass
+    # Remove contents from the bucket
+    bucket_name = os.environ.get("KCIDB_CACHE_BUCKET_NAME")
+    client = kcidb.cache.Client(bucket_name, 5 * 1024 * 1024)
```

Comment:
Please pass zero as the size limit here, as it doesn't matter in this case, makes it clear it doesn't matter, and we can avoid using a constant here later.

Reply:
okay